### DISCUSSION TITLE
Implementation of  the Vlare core .vlr format

### DISCUSSION THREAD

**Discussion by facehead** [sn4k3/UVtools#679] [Ideas]
Hello everyone, 
Just wondering if there is any chance for you to implement the Vlare core .vlr format?
Thanks in advance!

**Comment by sn4k3** [comment]
Being handled here: https://github.com/sn4k3/UVtools/issues/513
Need vlare to pass the spec to be implemented.

**Reply by facehead** [reply]
thanks so much, ill check it out and thanks for your work on Uv tools.
